City-of-Helsinki/liikunta-helsinki
# LIIKUNTA-222 | Add matomo tracking

## Description

Created Matomo component that handles user tracking

## Issues

### Closes

**[LIIKUNTA-222](https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-222):**

### Related

## Testing

### Automated tests

### Manual testing

With each page / url change there should be HTTP call to digiariis domain.

<img width="1680" alt="Screenshot 2022-02-01 at 9 38 22" src="https://user-images.githubusercontent.com/15219142/151934886-25c6623c-d9a1-4e99-939a-7434de8fdcf1.png">

## Screenshots



## Additional notes
